geode-connectors/src/main/java/org/apache/geode/connectors/jdbc/internal/JdbcConnectorService.java

```diff
@@ -31,6 +32,9 @@ void replaceRegionMapping(RegionMapping mapping)
 
   void destroyRegionMapping(String regionName);
 
+  boolean isRegionSynchronous(RegionMapping mapping, Cache cache)
```

Comment:
I think a better name would be "isMappingSynchronous". Under the covers it looks at the region to figure this out but callers of this method are really asking about the mapping they pass as the first parameter.